### PR TITLE
Goof around

### PR DESCRIPTION
**Pull request by Patater**
This change is [Reviewable](https://reviewable.io/reviews/alzix/try-reviewable-io/3)

### CODE REVIEW

**Comment by alzix** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[main.c, line 10 at r1](https://reviewable.io:443/reviews/alzix/try-reviewable-io/3#-KtBL5tLn8DA60fLbhHO:-KtBL5tLn8DA60fLbhHP:b-524sd7) ([raw file](https://github.com/alzix/try-reviewable-io/blob/809140f2f3aa6920a25fc7c2deee0f3e81c43bf0/main.c#L10)):*
> ```C
> 
> int banana = 0x00BA7A7A;
> extern int ding_dong;
> ```

who defines it?

---

*[README.md, line 4 at r1](https://reviewable.io:443/reviews/alzix/try-reviewable-io/3#-KtBKxXW0DUHBU1mcNED:-KtBKxXW0DUHBU1mcNEE:b-j07utr) ([raw file](https://github.com/alzix/try-reviewable-io/blob/809140f2f3aa6920a25fc7c2deee0f3e81c43bf0/README.md#L4)):*
> ```Markdown
> foofoo
> 
> How do you like this trailing whitespace?
> ```

seems it is just ignores it... :(

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/alzix/try-reviewable-io/3)*

**Comment by alzix** [comment]
Reviewed 1 of 2 files at r1.
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/alzix/try-reviewable-io/3)*

**Comment by Patater** [comment]
> seems it is just ignores it... :(


Well, maybe. I have added it in the first commit, and fixed it in a later commit. I was hoping to be able to see the problem in an earlier commit.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # try-reviewable-io
 foofoo
 
-
+How do you like this trailing whitespace?
```
Comment (Patater) [inline README.md]: This line has trailing whitespace.

**Comment by Patater** [comment]
I'm done goofing around.

**Pull request closed without merging**